docs/getting-started/sync-with-github.md

```diff
@@ -7,55 +7,171 @@ slug: /sync-with-github
 
 # Sync with GitHub
 
+Amplication can seamlessly synchronize with GitHub.
+Your application's auto-generated code gets pushed to a GitHub repository of your choosing.
```

Comment:
@dericksozo "The code generated by Amplication gets pushed to a GitHub repository of your choosing"